worlds/zillion/__init__.py

```diff
@@ -304,7 +304,8 @@ def finalize_item_locations(self) -> None:
         zz_patcher.all_fixes_and_options(zz_options)
         zz_patcher.set_external_item_interface(zz_options.start_char, zz_options.max_level)
         zz_patcher.set_multiworld_items(multi_items)
-        zz_patcher.set_rom_to_ram_data(self.world.player_name[self.player].replace(' ', '_').encode())
+        game_id = self.world.player_name[self.player].encode() + b'\x00' + self.world.seed_name[-6:].encode()
```

Comment:
seed_name can be unicode as well, so `self.world.seed_name[-6:].encode()` produces between 6 and 24 bytes - did you account for that?

Reply:
yup, 94 bytes available for the total